### PR TITLE
Python Strings

### PR DESCRIPTION
**Pull request by Piyush-Kumar99**
please review

### CODE REVIEW

**Comment by pranav1601** [comment]
Hey @Piyush-Kumar99 ,
Please put examples in examples folder,
Also, use "" to enclose strings.

File: datatypes/string.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    
+2. Accessing substring and concatenating it with another string
+    - var = "Pyflask"
+      var1 = var [:2] + 'thon' 
+      This initializez var1 with the value as "python" . (Note [:2] means starting from the first position (0th index) and terminating before 3rd element (2nd index) )
+    - x var[1:2] , var[5:10] ?
+
+3. escape charaters maybe used 
+    - non printable characters that can be represented by backslash notation
+    - ex \a,\b\ etc
```
Comment (pranav1601) [inline datatypes/string.md]: Please use "",
Example, instead of ex.
Or
eg: "\a", "\b" etc.

File: datatypes/string.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+                                                    Python strings.
```
Comment (pranav1601) [inline datatypes/string.md:1]: Write:
# Python Strings
Instead of 
` Python Strings.

Keep it professional

**Comment by Piyush-Kumar99** [comment]
Is it fine now @pranav1612000

File: datatypes/string.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    - str.lower()
+    - str.upper()
+    - str.strip()
+    - str.replace(old_str, new_str)   
+    - str.join(str)                                           (For ex. '_'.join('hello') gives 'h_e_l_l_o'
+                                                              )
+    - str.split()                                             (returns a list of strings that were separated by spaces in the string
+                                                               for ex. if i have a string A='we are delighted to see u' , then we get
+                                                               A= ['we','are','delighted','to','see','u
+                                                              )
```
Comment (pranav1601) [inline datatypes/string.md:36]: Please put examples in examples folder.

**Comment by Piyush-Kumar99** [comment]
Any more changes required?

**Pull request closed without merging**